Toolchain/BuildCMake.sh

```diff
@@ -22,6 +22,23 @@ fi
 
 [ -z "$MAKEJOBS" ] && MAKEJOBS=$($NPROC)
 
+check_sha() {
```

Comment:
The commit message for this change refers to "brew on PATH", but I think you mean "sha256sum on PATH"?

Reply:
sha256sum is installed via brew (via the coreutils package). I mean sha256sum, md5sum, `install -D` -- in short, stuff from brew.